rudder-webapp/SOURCES/Makefile

```diff
@@ -46,7 +46,7 @@ PROXY_ENV = $(if $(PROXY), http_proxy=$(PROXY))
 # Usage: $(GET) <destination_file> <url> <hash>
 # Pass an empty string to skip hash check
 # grep 'GET=' */SOURCES/Makefile to patch everywhere
-GET=get() { $(PROXY_ENV) curl  --silent --location --fail --output "$$1.part" "$$2" && { openssl dgst -sha256 "$$1.part" | grep -q "$$3" || { echo "Wrong checksum, aborting"; exit 1; }; } && mv "$$1.part" "$$1"; }; get
+GET=get() { $(PROXY_ENV) curl  --silent --cacert $(CURDIR)/letsencrypt-root-x1.pem --location --fail --output "$$1.part" "$$2" && { openssl dgst -sha256 "$$1.part" | grep -q "$$3" || { echo "Wrong checksum, aborting"; exit 1; }; } && mv "$$1.part" "$$1"; }; get
```

Comment:
should only be needed for rudder-agent as server OSes should work once docs.rudder.io uses ISRG X1 root.

Reply:
in 6.1 some server distro are in the "old" category